nucypher-core-python/src/lib.rs

```diff
@@ -720,6 +733,71 @@ impl ThresholdDecryptionRequest {
     }
 }
 
+//
+// E2EEThresholdDecryptionRequest
+//
+#[pyclass(module = "nucypher_core")]
+#[derive(derive_more::From, derive_more::AsRef)]
+pub struct E2EEThresholdDecryptionRequest {
+    backend: nucypher_core::E2EEThresholdDecryptionRequest,
+}
+
+#[pymethods]
+impl E2EEThresholdDecryptionRequest {
+    #[getter]
+    pub fn decryption_request(&self) -> ThresholdDecryptionRequest {
+        self.backend.decryption_request.clone().into()
```

Comment:
We should be able to skip `clone()` here, but `ThresholdDecryptionRequest` doesn't implement copy. `PublicKey` does, that's why we can just `self.backend.response_encrypting_key.into()` a few lines below.

Should we also derive `Copy` for `ThresholdDecryptionRequest`?

```suggestion
        self.backend.decryption_request.into()
```

Reply:
I don't think you can, since it has a `Box` field (and in general, stuff that has heap allocations inside cannot be `Copy`)